CONTRIBUTING.md

```diff
@@ -50,6 +50,12 @@ After cloning your fork, follow these steps to set up the project locally,
 4. **Do setup for projects by running:** 
       ```bash
       pnpm setup:project
+5. **In a terminal, start the dependencies (rabbitmq, mongodb and minio) as Docker containers by running the following command:**
+      ```bash
+      docker compose -f docker/dependencies.compose.yml up
```

Comment:
@Sidd-77 I suggest adding `-d` in `docker compose -f docker/dependencies.compose.yml up` so the developer can utilize the terminal for other work. Many don't know that closing the terminal or doing `CTRL + C` in the terminal will shut down containers if run without `-d`.